doc/classes/RichTextLabel.xml

```diff
@@ -666,6 +666,12 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="text_changed">
```

Comment:
This needs to move down, they must be sorted alphabetically

Reply:
Ahhh that explains why the action failed before. Fixed it